.gradletasknamecache

```diff
@@ -0,0 +1,2574 @@
+ideaModule
+ideaModule
+ideaModule
+ideaModule
+ideaModule
+jacocoMerge
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+javadocJar
+jmhReport
+jmhReport
+jmhReport
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+prepareKotlinBuildScriptModel
+processJmhResources
+processJmhResources
+processJmhResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+processTestResources
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+signArchives
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+sourcesJar
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
+verifyGoogleJavaFormat
```

Comment:
please remove this file, as @bogdandrutu said.

Reply:
Done, why not to put it in git ignore?

Reply:
If you want to do that, please make a separate PR. But we definitely don't want to merge that file :)